applications/minotari_console_wallet/src/ui/components/send_tab.rs

```diff
@@ -29,6 +29,7 @@ pub struct SendTab {
     send_input_mode: SendInputMode,
     show_contacts: bool,
     to_field: String,
+    payment_id_field: String,
```

Comment:
it might be good in future to include a drop down for "Custom Text", "Custom binary", "Sender Address"

Reply:
as well as some other options in future. Exchanges might want a specific field in here.